### PR TITLE
Use an aar to define bridge dependency

### PR DESCRIPTION
**Pull request by ahaisting-livefront**
## 📔 Objective

The goal here is to fix the failing `beta` variants on `main`. When defining `bridge` as a `project` dependency, a variant for `beta` cannot be found. Including an `.aar` as the dependency resolves this issue. 

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed
  issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -135,8 +135,8 @@ dependencies {
         add("standardImplementation", dependencyNotation)
     }
 
-    // TODO: this should use a versioned aar instead of referencing the project BITAU-94
-    implementation(project(":bridge"))
+    // TODO: this should use a versioned aar instead of referencing a local aar BITAU-94
+    implementation(files("libs/bridge-release.aar"))
```
[the remark this answers is not in the document]
Comment (ahaisting-livefront) [reply app/build.gradle.kts:139]: Yup, totally fair. 

I'm going to look into including that automatically on the bridge build side. I'm assuming that's what you mean.

File: bridge/README.md [hunk 2]
```diff
@@ -16,6 +17,14 @@
 - **Minimum SDK**: 28
 - **Target SDK**: 34
 
+## Building
+
+To build an aar for inclusion in consumer applications, run:
```
Comment (brian-livefront) [inline bridge/README.md:22]: `aar` should be `AAR` anywhere we refer to it.

File: app/build.gradle.kts [hunk 3]
```diff
@@ -135,8 +135,8 @@ dependencies {
         add("standardImplementation", dependencyNotation)
     }
 
-    // TODO: this should use a versioned aar instead of referencing the project BITAU-94
-    implementation(project(":bridge"))
+    // TODO: this should use a versioned aar instead of referencing a local aar BITAU-94
+    implementation(files("libs/bridge-0.1.0-release.aar"))
```
Comment (brian-livefront) [inline app/build.gradle.kts:139]: I'm still wondering if we should slap `snapshot` on here manually to make clear this isn't the _released_ version of `0.1.0`.

Comment (ahaisting-livefront) [reply app/build.gradle.kts:139]: I'll defer to you!

Our definition of _released_ is pretty nebulous. If its built and included in the source of the Bitwarden app, isn't it _released_?

I'm happy either way.

File: bridge/README.md (hunk 2 above)
Comment (ahaisting-livefront) [reply bridge/README.md]: Cool 👍

b650813527fc3cd7e866fb1841d0fa24cd0c1145

File: app/build.gradle.kts (hunk 3 above)
Comment (ahaisting-livefront) [reply app/build.gradle.kts]: I went ahead and added SNAPSHOT:

a02f475ab415274a596aef47d42f17c8a78d6b5b

**Review by brian-livefront — APPROVED** [review]

**Comment by ahaisting-livefront** [comment]
Thanks for the comments, @brian-livefront !

**Pull request merged**